Cargo.toml

```diff
@@ -49,3 +50,4 @@ default = ["mbed-crypto-provider", "pkcs11-provider"]
 mbed-crypto-provider = []
 pkcs11-provider = ["pkcs11", "picky-asn1-der", "picky-asn1"]
 tpm-provider = ["tss-esapi", "picky-asn1-der", "picky-asn1"]
+fuzz = ["parsec-interface/fuzz", "arbitrary"]
```

Comment:
Is the `parsec-interface/fuzz` feature still needed now? As we only pass a stream to the fuzzer and now longer a `Request` + `ApplicationName`?
I guess as discussed in #54 it will be used when fuzzing individual components, but for now should we remove it?

Reply:
yeah, i guess I can remove it and reintroduce when adding the other fuzzing stuff